### PR TITLE
Update prod IP address

### PR DESCRIPTION
**Pull request by morgolock**

### CODE REVIEW

File: CODIGO/FrmLogear.frm [hunk 1]
```diff
@@ -392,21 +392,16 @@ Private Sub cmdIngresar_Click()
...
             #Else
-                'Production and staging use this path
-                #If DEBUGGING = 0 Then
-                    Call SetDefaultServer
-                #Else
-                    IPdelServidorLogin = "45.235.98.31"
-                    PuertoDelServidorLogin = 11814
-                #End If
+                'When not in DEVELOPER mode we read the ip and port from the list
+                Call SetDefaultServer
```
Comment (matiascalegaris) [inline CODIGO/FrmLogear.frm:403]: why did you remove staging setting here?

Comment (morgolock) [reply CODIGO/FrmLogear.frm:403]: Because it's confusing that when I use DEBUG = 1 the client would not connect to prod. 

We need a new define `CONNECT_TO_STAGING` , `DEBUG` should not affect where the client connects to.

**Review by matiascalegaris — APPROVED** [review]

**Review by matiascalegaris — APPROVED** [review]
